components/automate-backend-deployment/habitat/bin/prepare-a2-deployment-workspace

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 echo "A2 deployment workspace will be created in ${PWD}/a2_deploy_workspace.  Please enter your sudo password if prompted"
-sudo rsync -avz  $(hab pkg path chef/automate-ha-deployment)/workspace/ a2_deploy_workspace
+sudo rsync -avz  $(hab pkg path schaudha/automate-ha-deployment)/workspace/ a2_deploy_workspace
```

Comment:
we should not change the origin from `chef` to your origin

Reply:
Hi @Dmaddu, the changes were made to test the terraform version after making changes in the code. I will revert it back to chef.